src/js/components/Box/doc.js

```diff
@@ -212,7 +212,9 @@ export const doc = Box => {
       PropTypes.string,
     ]).description(`The amount of spacing between child elements. This
         should not be used in conjunction with 'wrap' as the gap elements
-        will not wrap gracefully.`),
+        will not wrap gracefully. If a child component renders a fragment,
+        Box will not add a gap between the individual elements in the
+        fragment.`),
```

Comment:
```suggestion
        will not wrap gracefully. If a child is a Fragment,
        Box will not add a gap between the children of the Fragment.`),
```

Reply:
This might get the point across but it's not the situation in issue that was submitted. In this issue the child is not a fragment. It's a functional component that renders a fragment. It actually produces a different component tree, at least from what Box can figure out when it's rendered.

Reply:
Thanks for the quick call, I think we can incorporate the feedback and wrap up this issue.

Reply:
Simplified the wording